eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Plutus/Evaluate.hs

```diff
@@ -350,9 +355,9 @@ evalTxExUnits ::
   --  sufficient execution budget.
   --  Otherwise, we return a 'TranslationError' manifesting from failed attempts
   --  to construct a valid execution context for the given transaction.
-  Either (ContextError era) (RedeemerReport era)
-evalTxExUnits pp tx utxo ei sysS =
-  Map.map (fmap snd) <$> evalTxExUnitsWithLogs pp tx utxo ei sysS
+  RedeemerReport era
+evalTxExUnits pp tx utxo epochInfo systemStart =
+  Map.map (fmap snd) $ evalTxExUnitsWithLogs pp tx utxo epochInfo systemStart
 
 -- | Evaluate the execution budgets needed for all the redeemers in
 --  a given transaction. If a redeemer is invalid, a failure is returned instead.
```

Comment:
```suggestion
-- | Evaluate the execution budgets needed for all the redeemers in
--  a given transaction.
```

Reply:
Same comment about the comment above for `evalTxExUnits` here: https://github.com/IntersectMBO/cardano-ledger/pull/4374/files#diff-d42005a7dd7c31638237a88fcf0ffe4feb1cca752f80b3d8901d8626e6d10f45R334